deploy/backplane/mkrts/redhat-kas-fleetshard/20-mkrts-fleetshard-admins.SubjectPermission.yml

```diff
@@ -1,12 +1,14 @@
 apiVersion: managed.openshift.io/v1alpha1
 kind: SubjectPermission
 metadata:
-  name: backplane-mkrts-admins
+  name: backplane-mkrts-users
   namespace: openshift-rbac-permissions
 spec:
+  clusterPermissions:
+  - backplane-mkrts-users-project
```

Comment:
This will create a clusterrolebinding, do you need this?

Reply:
Yeah, I believe we are already creating the binding we want on line 9-12 in this file. I think we misunderstood the clusterPermissions block.